api_v3/lib/types/entry/KalturaLiveEntry.php

```diff
@@ -274,4 +298,32 @@ protected function validateRecordingOptionsChanged($sourceObject, $attr)
 			$this->validateRecordingDone($sourceObject, "recordingOptions");
 		}
 	}
+
+	protected function validateSegmentDurationChanged($sourceObject, $attr)
```

Comment:
typo... the call to this function should have been from validateForUpdate() line 216.
code fix: "segmentDuration" => array("validatePropertyChanged", "validateRecordingOptionsChanged"),